# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/TransformFeedback11.cpp
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/TransformFeedback11.cpp
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/TransformFeedback11.cpp	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/TransformFeedback11.cpp	Sat Sep 07 13:38:36 2019 +0000
@@ -51,7 +51,7 @@
 angle::Result TransformFeedback11::end(const gl::Context *context)
 {
     mRenderer->getStateManager()->invalidateTransformFeedback();
-    if (mRenderer->getWorkarounds().flushAfterEndingTransformFeedback)
+    if (mRenderer->getFeatures().flushAfterEndingTransformFeedback.enabled)
     {
         mRenderer->getDeviceContext()->Flush();
     }
@@ -70,13 +70,6 @@
     return angle::Result::Continue;
 }
 
-angle::Result TransformFeedback11::bindGenericBuffer(const gl::Context *context,
-                                                     const gl::BindingPointer<gl::Buffer> &binding)
-{
-    mRenderer->getStateManager()->invalidateTransformFeedback();
-    return angle::Result::Continue;
-}
-
 angle::Result TransformFeedback11::bindIndexedBuffer(
     const gl::Context *context,
     size_t index,